language_server/src/lsp/feature_language_features.cpp

```diff
@@ -402,13 +452,50 @@ json feature_language_features::document_symbol_list_json(
 
 void feature_language_features::document_symbol(const json& id, const json& params)
 {
-    auto document_uri = params["textDocument"]["uri"].get<std::string>();
+    auto document_uri = extract_document_uri(params);
 
     const auto limit = 5000LL;
     auto symbol_list = ws_mngr_.document_symbol(document_uri.c_str(), limit);
 
     response_->respond(id, "", document_symbol_list_json(symbol_list));
 }
 
+void feature_language_features::opcode_suggestion(const json& id, const json& params)
```

Comment:
Do you plan to add the suggestions to the completion list as well? I think it might be even more handy there since the user could see the more options.

E.g. When I type `AAA` I could be glad to see other options than `ADATA` like the suggested ones `LAA`, `LA` and others

Reply:
that sound like a good idea, maybe a follow up PR?

Reply:
Sure!